src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="searchBar largeSearchBar" data-qa="big">
```

Comment:
Due to the [checklist ](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)so no need to create `largeSearchBar` class, you can move similar classes to `searchBar` class and add some additional styles to your small form by adding additional class `searchBar--small` to your small form

![image](https://user-images.githubusercontent.com/80700247/229345132-93e43cf1-9532-4b40-8549-bc2cf9544f59.png)


Apply this everywhere